src/move/econia/sources/market.move

```diff
@@ -6086,6 +6373,68 @@ module econia::market {
             swap_coins(market_id, integrator, direction_taker, min_base,
                        max_base, min_quote, max_quote, price, base_coins,
                        quote_coins);
+        let taker_order_id = order_id_no_post(2);
+        // Assert events.
+        assert!(user::get_place_limit_order_events_test(
+            market_id, maker_address, custodian_id) == vector[
+                user::create_place_limit_order_event_test(
+                    market_id,
+                    maker_address,
+                    custodian_id,
+                    @integrator,
+                    side_maker,
+                    size_maker,
+                    price,
+                    restriction,
+                    self_match_behavior,
+                    size_maker,
+                    market_order_id
+                )
+            ], 0);
+        assert!(user::get_fill_events_test(
+            market_id, maker_address, custodian_id) == vector[
+                user::create_fill_event_internal(
+                    market_id,
+                    size_taker,
+                    price,
+                    side_maker,
+                    maker_address,
+                    custodian_id,
+                    market_order_id,
+                    NO_TAKER_ADDRESS,
+                    NO_CUSTODIAN,
+                    taker_order_id,
+                    fee,
+                    0
+                )
+            ], 0);
+        assert!(user::get_cancel_order_events_test(
+            market_id, maker_address, NO_CUSTODIAN) == vector[], 0);
+        assert!(get_place_swap_order_events_market_test(
+            market_id) == vector[
+                PlaceSwapOrderEvent{
```

Comment:
Why no helper for these other events? (advocating for consistency here)

Reply:
Unlike the structs from https://github.com/econia-labs/econia/pull/366#discussion_r1277970860, the `PlaceSwapOrderEvent` is only ever emitted to an event handle that is define in the market module

Hence it can be created directly here using curly brace packing syntax